app/views/provider/debt-management/990175001.html

```diff
@@ -0,0 +1,314 @@
+<!-- Use this page as the index for your project -->
+
+<!-- ADDING CUSTOM CSS - Add your custom CSS or Sass in /app/assets/sass/main.scss -->
+
+<!-- Extends the layout from /views/layout.html -->
+{% extends 'default-layout.html' %}
+
+<!-- Set the page title -->
+{% block pageTitle %}
+Contract details
+{% endblock %}
+
+<!-- Breadcrumbs -->
+{% block beforeContent %}
+{{ breadcrumb({
```

Comment:
Indent section.. check all your files and indent as its a repeated issue.

Reply:
fixed